sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/IProgressReceiver.java

```diff
@@ -8,10 +8,10 @@
 /**
  * An {@code IProgressReceiver} is an object that can be used to report progress on network transfers. When specified on
  * transfer operations, the {@code reportProgress} method will be called periodically with the total number of bytes
- * transferred. The user may configure this method to report progress in wha tever format desired. It is recommended
+ * transferred. The user may configure this method to report progress in whatever format desired. It is recommended
  * that this type be used in conjunction with {@link ProgressReporter#addProgressReporting(Flux, IProgressReceiver)}.
  */
-interface IProgressReceiver {
+public interface IProgressReceiver {
```

Comment:
I'm not a fan of interfaces starting with `I`. I would much prefer to see this called `ProgressReceiver`.

Reply:
do we have any info about this in out guidance?  for the preferred prefix-naming stuff?  

I see a few interfaces doing the `I..` naming within the project
![image](https://user-images.githubusercontent.com/24213737/66438722-66e9f480-e9e2-11e9-9ccb-49299edbc342.png)

Might be a good idea to have a docs with any agreement on this.

I don't really have any specific preference regarding to this.  I am fine updating it :)

Reply:
Yes, I will update spec to say to not do this.

Reply:
I don't think there is any public API with the I prefix. If you do find any, please let me know.